src/scss/templates/_front-page.scss

```diff
@@ -5,3 +5,7 @@
     color: $brand-default-aw;
   }
 }
+
+.home-hr-lighter {
```

Comment:
This will work for the homepage, but we may want to consider overriding the default `hr` border color instead of using a unique homepage class to help establish consistency with other dividers on other pages (e.g. category pages)

Reply:
I thought about overriding the `hr` border-color, but it sounded like the request was specifically for the homepage; I'm not sure we want all `hr`s to be this light of a color.